tools/triage/clean_old_issues.py

```diff
@@ -106,6 +108,11 @@ def update_issues(jira_client: jira.JIRA, issues: list[jira.Issue], logger: logg
         raise ExceptionGroup("Failed to update all issues.", errors)
 
 
+@retry(exceptions=jira.JIRAError, tries=3, delay=1)
+def apply_function_with_retry(function: Callable, *args, **kwargs):
+    function(*args, **kwargs)
```

Comment:
```suggestion
    return function(*args, **kwargs)
```
else ``result_list = apply_function_with_retry(...`` shouldn't work

Reply:
my bad